src/components/Search/NavigationItem/NavigationItem.module.scss

```diff
@@ -1,17 +1,7 @@
 .container {
-  display: flex;
+  display: inline-flex;
```

Comment:
This file doesn't seem to be used at all. If we don't need it, we should remove it.